packages/core/src/input/input.js

```diff
@@ -3,14 +3,16 @@ import { jsx } from "@emotion/core";
 import React from "react";
 
 export const InputIconRenderer = () => <>🔍</>;
+export const ClearIconRenderer = () => <>✘️</>;
```

Comment:
Do these two need the Fragment wrapping? I think it's just like any other string - a valid component.

Reply:
I tried but it didn't work :)